src/ui/views/AccessMyAccount.vue

```diff
@@ -294,8 +294,14 @@ export default defineComponent({
             try {
                 const { Ed25519PrivateKey, Mnemonic } = await import(/* webpackChunkName: "hashgraph" */ "@hashgraph/sdk");
 
+                const mnemonic = new Mnemonic(accessByPhraseState.words);
+
+                if (!mnemonic.validate().isOk()) {
+                    throw new Error("Invalid Mnemonic");
```

Comment:
This cannot actually error. People need to be _allowed_ to use an invalid mnemonic due to us not denying it in the past.